src/scroll-animation.js

```diff
@@ -138,6 +138,14 @@ export default class ScrollAnimation extends Component {
       || previousVis.onScreen !== currentVis.onScreen
   }
 
+  captureRef(ref) {
+    if (ref) {
+      this.node = ref.animateRef
+        ? ref.animateRef
+        : ref
```

Comment:
this doesn't exactly matter, but I was wondering if this would provide any reading value and still be correct?
```js
this.node = ref.animateRef || ref
```